api-reference/10 UI Components/dxTabs/1 Configuration/items/items.md

```diff
@@ -4,3 +4,19 @@ type: Array<String, dxTabsItem, any>
 firedEvents: optionChanged
 inheritsType: dxTabsItem
 ---
+The **items** array can contain:
+
+- Strings
+- Objects with fields described in this section
```

Comment:
It seems a bit odd to mention `fields described in this section` and then also `any other fields`

```suggestion
- Objects with fields described in this section. If you want use other fields, specify the [itemTemplate](/api-reference/10%20UI%20Components/CollectionWidget/1%20Configuration/itemTemplate.md '{basewidgetpath}/Configuration/#itemTemplate').
```